### DISCUSSION TITLE
I want to Contribute

### DISCUSSION THREAD

**Discussion by ghost** [Dev-Soumyaranjan/Java-Coding-Questions#24] [Q&A] [answered]
I am new to Git-hub can any one guide me through I know decent java and want to contribute to this repo.

**Comment by Dev-Soumyaranjan** [comment, marked as answer]
> I am new to Git-hub can any one guide me through I know decent java and want to contribute to this repo.

#### Welcome to our repo
First of all thank you so much for considering to contribute to our small initiative for java beginners.
I have added some resources in the [README.md](https://github.com/Dev-Soumyaranjan/Java-Coding-Questions/blob/main/README.md) to learn fundamentals of GitHub which will surely help you to make first pull request for your open source journey.

Otherwise you can precisely use our [How-to-Contribute.md](https://github.com/Dev-Soumyaranjan/Java-Coding-Questions/blob/main/How-to-Contribute.md) (Recommended for now) file to get started.

And if at any point you got stuck then without hesitation you can write your query in discussion page. Also you can contact me through my mail id dev.soumya369@gmail.com
